src/main/java/io/vertx/kafka/admin/KafkaAdminClient.java

```diff
@@ -180,6 +183,68 @@ static KafkaAdminClient create(Vertx vertx, Properties config) {
    */
   Future<Map<String, ConsumerGroupDescription>> describeConsumerGroups(List<java.lang.String> groupIds);
 
+  /**
+   * Delete consumer groups from the cluster.
+   *
+   * @param groupIds the ids of the groups to delete
+   */
+  void deleteConsumerGroups(List<java.lang.String> groupIds, Handler<AsyncResult<Void>> completionHandler);
```

Comment:
way using the package + class name and not just `String`?

Reply:
I am mystified where those came from. I have removed the package name.